mathics/builtin/inout.py

```diff
@@ -1783,7 +1783,9 @@ def apply_mathml(self, expr, evaluation):
             xml = ''
         # mathml = '<math><mstyle displaystyle="true">%s</mstyle></math>' % xml
         # #convert_box(boxes)
-        mathml = '<math>%s</math>' % xml  # convert_box(boxes)
+        if True:  # configuration to be defined
+            xml = '<mstyle mathvariant="sans-serif">%s</mstyle>' % xml
```

Comment:
can we add this as a settings option. Perhaps `XML_FONT = None` by default.

Reply:
@poke1024 I belive this is the only thing preventing us from merging the PR (?)

Reply:
@GarkGarcia @sn6uv From a user standpoint, what is a "settings option" and how does one add this? 

I like this feature.

Reply:
Please see PR #908